osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -30,31 +34,37 @@ public DrawableJudgement(Judgement judgement)
         {
             Judgement = judgement;
 
-            AutoSizeAxes = Axes.Both;
-
-            Children = new[]
-            {
-                JudgementText = new OsuSpriteText
-                {
-                    Origin = Anchor.Centre,
-                    Anchor = Anchor.Centre,
-                    Text = judgement.Result.GetDescription().ToUpper(),
-                    Font = @"Venera",
-                    Scale = new Vector2(0.85f, 1),
-                    TextSize = 12
-                }
-            };
+            Size = new Vector2(judgement_size);
         }
 
         [BackgroundDependencyLoader]
         private void load(OsuColour colours)
         {
+            string legacyName = string.Empty;
             switch (Judgement.Result)
             {
                 case HitResult.Miss:
-                    Colour = colours.Red;
+                    legacyName = "hit0";
+                    break;
+                case HitResult.Meh:
+                    legacyName = "hit50";
+                    break;
+                case HitResult.Good:
+                    legacyName = "hit100";
+                    break;
+                case HitResult.Great:
+                    legacyName = "hit300";
                     break;
             }
+
+            Child = new SkinnableDrawable($"Play/osu/{legacyName}", _ => JudgementText = new OsuSpriteText

```

Comment:
How about instead exposing a `virtual string GetResourceName(HitResult result) => "Play/osu/{result.ToString().ToLower()";`? Two reasons:
1. Eventually we're not going to have `Play/osu/`. When we move these resources to each gamemode they should exist as simply `Play/`.
2. Will allow us to override this lookup further depending on additional construction parameters (eg, per-key judgements in mania).